Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM joshmahowald/cloud-workstation AS baseworkstation
+
+
+
+FROM ruby:alpine
+RUN apk --no-cache --update add bash  \
+  ca-certificates openssl openssh && update-ca-certificates  
+
+
+ENV PROJ_DIR=/usr/local/src/kitchen-terraform
+WORKDIR $PROJ_DIR
+ADD Gemfile .
```

Comment:
The image should be built based on the latest version of Kitchen-Terraform published to RubyGems to ensure consistency in behaviour.

Reply:
I was actually thinking that the image would just be built from the local source.  Can easily have docker hub then just build a new image on every release with an automated build, including all your tag information.  I guess though I don't know enough about when/how ruby gems is updated.